src/storage/storage.h

```diff
@@ -243,7 +245,7 @@ class Storage {
   Config *config_ = nullptr;
   std::vector<rocksdb::ColumnFamilyHandle *> cf_handles_;
   LockManager lock_mgr_;
-  bool db_size_limit_reached_ = false;
+  std::atomic<bool> db_size_limit_reached_{false};
```

Comment:
Why change it to atomic? I don't quite understand.

Reply:
This variable will be modified by multiple threads during compact and flush.